src/store/modules/topic.ts

```diff
@@ -2,9 +2,12 @@ import { ActionContext } from "vuex";
 import { State as RootState } from "@/store";
 import { ITopic } from "@/interfaces/api/v2/topic.interface";
 import Topic from "@/services/api/v2/topic.service";
+import V4Topic from "@/services/api/v4/topic.service";
```

Comment:
Can import like `import Topic as V4Topic from "@/services/api/v4/topic.service"`. Eventually when V2 gets deprecated we don't have to change the name at the class level.